ddl/ddl_api.go

```diff
@@ -5233,7 +5233,7 @@ func extractTblInfos(is infoschema.InfoSchema, oldIdent, newIdent ast.Ident, isA
 		if tableExists(is, newIdent, tables) {
 			return nil, 0, infoschema.ErrTableExists.GenWithStackByArgs(newIdent)
 		}
-		return nil, 0, errFileNotFound.GenWithStackByArgs(oldIdent.Schema, oldIdent.Name)
+		return nil, 0, infoschema.ErrTableNotExists.GenWithStackByArgs(oldIdent.Schema, oldIdent.Name)
```

Comment:
there is another usage at L5227, can you please also change it?

Reply:
sure I can. before did not change it cause new to code need more time more familar, so try change less, and at then not sure if the message same better in the case. I will change L5227 and pull again

Reply:
@xiongjiwei I also changed L5227, and use "push -f" forced to commit the new change, please review

Reply:
great! thank you for your contribution

BTW, there are some failed tests in CI, you can click the link below to find it, the `check_dev_2` fail seems not related to your changes, just ignore it.

Reply:
Thanks! Anything need, just let me know.

Reply:
any update for the fail tests? you need to update the tests

Reply:
I forced to commit change, can you see the new changes?

Reply:
It seem my new change not commited successfully on this branch

Reply:
no, I can only see one commit and there are no other changes after I reviewed last time

Reply:
@xiongjiwei I finally commit the new change, please review. 
I changed more lines for "_test.go" to make test correct. Actually, its more easy just to change to ErrFileNotFound‘s message same as ErrNoSuchTable, but I thought that will make this message "file not found..." disappear for later usage, so for long run, seem compatible mysql8.0 message and more change is better.